tools/cd_scripts/package_gcsfuse.sh

```diff
@@ -80,20 +84,21 @@ sudo apt-get install git -y
 # It is require for multi-arch support
 sudo apt-get install qemu-user-static binfmt-support
 
-git clone https://github.com/GoogleCloudPlatform/gcsfuse.git
+# TODO (raj-prince): to remove this branch once read-cache-release branched is merged into the master branch.
+git clone -b "read_cache_release" https://github.com/GoogleCloudPlatform/gcsfuse.git
```

Comment:
```suggestion
git clone https://github.com/GoogleCloudPlatform/gcsfuse.git
git checkout <commit hash from details.txt>
```

Reply:
My bad, just realised that we create details.txt after building the package. We will have to move that stage prior or pass commit hash as metadata to the VM.

Reply:
Discussed offline that we will do this later to unblock the beta release.